### PR TITLE
added a game scene

### PR DESCRIPTION
**Pull request by vultouri03**
added the scene in which the game will take place

### CODE REVIEW

File: src/js/game.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import '../css/style.css'
 import { Actor, Engine, Vector } from "excalibur"
```
Comment (soupremeleader) [inline src/js/game.js:2]: Actor, Vector and Resources are unused

File: src/js/GameScene.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Actor, Engine, Scene, Vector } from "excalibur"
+
```
Comment (soupremeleader) [inline src/js/GameScene.js:2]: imports except for Scene are unused

File: src/js/game.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
 import { Resources, ResourceLoader } from './resources.js'
```
Comment (soupremeleader) [inline src/js/game.js]: imports other than ResourceLoader, Engine and Gamescene are unused

File: src/js/GameScene.js (hunk 2 above)
Comment (vultouri03) [reply src/js/GameScene.js:2]: the unused imports have been removed

File: src/js/game.js (hunk 3 above)
Comment (vultouri03) [reply src/js/game.js]: the unnecessary imports have been removed

**Review by soupremeleader — APPROVED** [review]

**Review by SemvanderStigchel — APPROVED** [review]

**Pull request merged**